packages/wallet/frontend/src/pages/card/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { AppLayout } from '@/components/layouts/AppLayout'
+import { PageHeader } from '@/components/PageHeader'
+import { CardButtonMenu } from '@/components/userCards/CardButtonMenu'
+import { UserCard } from '@/components/userCards/UserCard'
+import { useCardContext } from '@/lib/context/card'
+import { NextPageWithLayout } from '@/lib/types/app'
+
+const UserCardPage: NextPageWithLayout = () => {
+  const { cardType } = useCardContext()
+  return (
+    <>
+      <div className="flex items-center justify-between md:flex-col md:items-start md:justify-start">
+        <PageHeader title="Your Card" />
+      </div>
+      <div className="flex justify-center items-center flex-col gap-6">
+        <UserCard type={cardType} />
```

Comment:
I don't think we should keep the state for the card - 'normal'/'details'. We should not show the card details directly. If the user goes to the card page, checks the details, moves to another page and comes back, the card details will be displayed directly.